cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -524,6 +533,80 @@ func getMultilinePrefix(currentPrefix string) string {
 	return "?"
 }
 
+// getRootMultiLineObjectPrefix generates the multiline prefix for an object.
+func getRootMultiLineObjectPrefix(obj ctrlclient.Object, objectTree *tree.ObjectTree) string {
+	// If the object is the last one in the tree, no prefix is needed.
+	if ensureLastObjectInTree(obj, objectTree) == string(obj.GetUID()) {
+		return ""
+	}
+
+	// Determine the prefix for the current object.
+	// If it is a leaf we don't have to add any prefix.
+	var prefix string
+	if len(objectTree.GetObjectsByParent(obj.GetUID())) > 0 {
+		prefix = pipe
+	}
+
+	// Traverse upward through the tree, calculating each parent's prefix.
+	// The parent of the root object is nil, so we stop when we reach that point.
+	previousUID := obj.GetUID()
+	parent := objectTree.GetParent(obj.GetUID())
+	for parent != nil {
+		// We have to break the loop if the previous ID is the same as the current ID.
+		// This should never happen as the root object doesn't have set the parentship.
+		if previousUID == parent.GetUID() {
+			break
+		}
```

Comment:
@fabriziopandini added this condition in my latest push to ensure that we never have an infinite loop. This should never happen, just to prevent that someone sets the root ID as parentship for the root object.